Etabs_Adapter/ETABSAdapter.cs

```diff
@@ -60,6 +60,7 @@ public partial class ETABSAdapter : BHoMAdapter
         public const string ID = "ETABS_id";
 
         public string FilePath { get; set; }
+        public string etabsVersion;
```

Comment:
Please change this to one of the following:

Option 1:
```suggestion
        private string m_EtabsVersion;
```

and move it down to the private fields (the m_app, m_model etc)

Option 2:
If you want it available in the UI, then make it a property (getter and setter) and capital letter
```suggestion
        public string EtabsVersion { get; set; }
```